pygama/lgdo/vectorofvectors.py

```diff
@@ -94,3 +96,20 @@ def set_vector(self, i_vec, nda):
             self.flattened_data.nda.resize(2*len(self.flattened_data.nda))
         self.flattened_data.nda[start:end] = nda
         self.cumulative_length.nda[i_vec] = end
+
+    def tolist(self):
+        l = []
+        l.append(list(self.flattened_data[:self.cumulative_length[0]]))
+        l += [list(self.flattened_data[self.cumulative_length[i-1]:self.cumulative_length[i]]) for i in range(1, len(self.cumulative_length))]
```

Comment:
This uses an explicit loop, which could be slow. I wonder if implementing __iter__() might be better. I'm also not remembering why we didn't use awkward arrays for VofV